### ISSUE TITLE
Update Delivery School Epic document with insights from the last ticket

### ISSUE THREAD

**Issue by katefisher808** [18F/TLC-crew#86]
Project Name
Update the [Delivery School Epic document](https://docs.google.com/document/d/1UWyX4E26BOFaDCt_nP_tIMjDwPzvjq-dW4_YGOttoaY/edit#heading=h.5ascrv66w4uo) template and context to capture insights from the work of the [last ticket](https://github.com/orgs/18F/projects/41/views/1?pane=issue&itemId=20811911)

A brief description of the work

Billable?
No

Acceptance Criteria
Kelsey, as epic owner, is aware of the epic document template changes 
Kelsey, as epic owner, is aware and agrees with the additional context/insights
The previous team who worked on the last ticket is aware and agrees with the additional context/insights

Skills needed
 Any human

**Comment by katefisher808** [comment]
I updated the epic document with initial responses to open questions from the work of ticket 38.
https://docs.google.com/document/d/1UWyX4E26BOFaDCt_nP_tIMjDwPzvjq-dW4_YGOttoaY/edit#

**Comment by katefisher808** [comment]
Review request in slack -
For review, it would be great to have one of the collaborators on ticket 38 to scan the responses I put in the open questions section for edits or additions. I was not able to answer all the questions.

**Comment by katefisher808** [comment]
@kfoley-18F 
Closing this ticket out - We made some edits to the [DS epic document](https://docs.google.com/document/d/1UWyX4E26BOFaDCt_nP_tIMjDwPzvjq-dW4_YGOttoaY/edit#) + added responses to the bottom based on our conversations with you. The hope is that the document can act as a repository for a collection of tickets and help with the onboarding of different members. If you have time, take a look! there are some open comments that the future owner of this work would ideally resolve.
